data/data/bootstrap/files/usr/local/bin/bootkube.sh.template

```diff
@@ -136,6 +136,8 @@ then
 
 	rm --recursive --force auth-api-bootstrap
 
+	CLUSTER_PROFILE_ANNOTATION="self-managed-high-availability"
```

Comment:
I wonder if it makes more sense to move the original definition of this variable on line 103 out of the conditional and deduplicate? 

It's only two instances in the same file so I'm not too concerned about this.

/approve

The original comment says:

>   # A more desireable end state would be require wiring up the infrastructure in some way and read it from rendered-manifest-dir

So this annotation is in manifests rendered during the api bootstrap phase?

Reply:
So this input to the flag would be use to filter the CRD manifests output in this stage to be only ones that map to the cluster profile specified.